instruqt/custom-fields/03-updating-the-application/assignment.md

```diff
@@ -18,60 +21,103 @@ tabs:
+
+<p align="center"><img src="../assets/cf-set-vars.png" width=600></img></p>
+
+If you receive the same result then we have our dev environment ready!
+
+## Downloading the Application Manifests
 
-You should see results similar to this:
+Next, we are going to download the manifests from the vendor portal. To download the latest version of the application, we'll employ the [release download](https://docs.replicated.com/reference/replicated-cli-release-download) command.
 
+The command needs the `SEQUENCE` we want to download, which was displayed in the results of the `release ls` command:
 ```
 SEQUENCE    CREATED                 EDITED                  ACTIVE_CHANNELS
 1           2022-09-20T19:53:57Z    0001-01-01T00:00:00Z    stable
 ```
-
-We want to update the release currently on the **CustomFields** channel. Note the **SEQUENCE** associated to that channel as that is what we are going to use in the next command.
-
 Let's create a directory structure before we start dowloading files. Create a directory for this lab in your environment and a `manifests` sub directory to store the manifests.
 
 ```
-mkdir custom-fields-track
+mkdir custom-fields-app
 
-cd custom-fields-track
+cd custom-fields-app
 
 mkdir manifests
 
 ```
 To download the contents of the release run the following command
 
 ```
-replicated release download [The SEQUENCE number from above] -d ./manifests
+replicated release download 1 -d ./manifests
 
 ```
-
-As an exmaple, if I wanted to download the manifests associated to the release in the **CustomFields** example above, I would run:
+A succesful download will yield results similar to this:
+
+```shell
+  • Fetching Release 1 ✓  
+  • Writing files to ./manifests
+    • k8s-app.yaml
+    • kots-app.yaml
+    • kots-support-bundle.yaml
+    • nginx-deployment.yaml
+    • nginx-feature-off.yaml
+    • nginx-service.yaml
 
 ```
-replicated release download 1 -d ./manifests
-```
 
-We are going to add a second `ConfigMap` that will be used when the Super Duper Feature is enabled.
+Now we are ready to update the application.
+
+## Making Updates to the Application
+
+We are going to add a second `ConfigMap` that will be used when the Super Duper Feature is enabled. 
+
+Head over to the **Code Editor** tab, which contains a director tree that shouls look similar to this
 
-Create a new file in the **./manifests** directory called `cp-feature-on.yaml` with the following content
+<p align="center"><img src="../assets/cf-code-edit-init.png" width=450></img></p>
+
+You can expand the **custom-fields-app** directory to see its contents:
+
+<p align="center"><img src="../assets/cf-app-dir.png" width=350></img></p>
+
+Use the **New File** icon (highlighted in red below) to create a new file in the **manifests** directory
+
+<p align="center"><img src="../assets/cf-new-file-icon.png" width=350></img></p>
+
+Enter `cp-feature-on.yaml` as the file name in the dialog 
```

Comment:
Should this be `nginx-featue-on.yaml` like in the screenshot?

Reply:
Yes and have made a new commit to reflect the correct file name